.github/workflows/verify.yaml

```diff
@@ -0,0 +1,76 @@
+name: verify
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+
+      - name: Install ShellCheck and shfmt
+        run: |
+          sudo apt update
+          sudo apt install -y shellcheck
+          go install mvdan.cc/sh/v3/cmd/shfmt@latest
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: latest
+          only-new-issues: true
+          args: --timeout=5m
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: codespell
+        uses: codespell-project/actions-codespell@v2
+        with:
+          skip: vendor,*.svg
+      
+      - name: ShellCheck and shfmt
+        run: |
+          shellcheck ./test/*.sh
+          shfmt -d ./test/*.sh
+
+  gomod:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+
+      - name: verify go.mod/go.sum
+        run: |
+          make vendor
+          git diff --exit-code
+
+  lint_markdown:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          base: 'main'  
+          filters: |
+            md:
+              - 'README.md'
+              - '.github/workflows/verify.yml'
+
+      - name: Lint markdown
+        if: steps.changes.outputs.md == 'true'
+        uses: DavidAnson/markdownlint-cli2-action@v10
+        with:
+          globs: |
+            README.md
+            .github/workflows/verify.yml
```

Comment:
It looks like [`glob`](https://github.com/DavidAnson/markdownlint-cli2-action?tab=readme-ov-file#globs-optional) is used to specify Markdown files to lint. Why is `verify.yml` added here?

Reply:
It's not really necessary I think I just add it thinking it may contain Markdown content that needs to be linted.

Reply:
@rst0git I think it was added to allow the job to be triggered when the workflow file itself is changed, even if no other markdown files are changed in the commit.

Reply:
Never mind, I just realised I was looking at the job above this (the paths filtering one). My bad